specification/maps/data-plane/Microsoft.Maps/Weather/preview/1.0/weather.json

```diff
@@ -76,6 +76,13 @@
     }
   },
   "parameters": {
+    "Pollutants": {
+      "name": "Pollutants",
+      "in": "query",
+      "type": "string",
```

Comment:
the description mentions this is boolean?

Reply:
it'll work either way as our web server will translate the string value to boolean when we hydrate the request query object but swagger does support a boolean type so we can use that as well. See if we have any other API that has a boolean and if we use type = string or boolean. As i said it'll work either way at the service level so atleast we can be consistent

Reply:
booleans are way better for generated clients

Reply:
@sataneja: I think we should update others to bool too in another PR for consistency.

I mean, generated code is like:

```go
alertsResp, err := weatherClient.GetSevereWeatherAlerts(ctx, weather.ResponseFormatJSON, "48.057,-81.091", &weather.WeatherGetSevereWeatherAlertsOptions{
   Details:  to.StringPtr("true"),
   Language: to.StringPtr("EN"),
})
```
Having string parameters for bools is weird from clients perspective. Generated bools are better:

```go
currentAirQualityResp, err := weatherClient.GetCurrentAirQuality(ctx, weather.ResponseFormatJSON, "47.632346,-122.138874", &weather.WeatherGetCurrentAirQualityOptions{
    Language:   to.StringPtr("EN"),
    Pollutants: to.BoolPtr(true),
})
```